precommit/core.d/patchfiles.sh

```diff
@@ -327,9 +327,10 @@ function patchcmd_apply
 function patchfile_apply_driver
 {
   declare patchfile=$1
+  declare gpg=$2
 
   if declare -f ${PATCH_METHOD}_apply >/dev/null; then
-    "${PATCH_METHOD}_apply" "${patchfile}"
+    "${PATCH_METHOD}_apply" "${patchfile}" "${gpg}"
```

Comment:
So i was looking some more into this.
I was thinking that if I rename ```{gpg}``` to ```{extra_args}```.
With this, gitapply_apply will work as it is currently.
And in patchcmd_apply will not work since the extra_args will not be processed and can be hooked in later if needed.

Does this make sense?